components/admin-dashboard/profile.jsx

```diff
@@ -0,0 +1,89 @@
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import PropTypes from 'prop-types';
+
+const getInterviews = (applications) => {
+  let interviews = [];
+  applications.forEach(application => {
+    const nextInterview = application.interviews.data.length - 1;
+    const interview = {
+      _id: application.interviews.data[nextInterview]._id,
+      company: application.company,
+      role: application.role,
+      type: application.interviews.data[nextInterview].type,
+      date: application.interviews.data[nextInterview].date
+    };
+    interviews.push(interview);
+  })
+  return interviews;
+};
+
+toast.configure();
+export default function Profile({ entry }) {
+  const sendEmailSuccessToast = () => {
```

Comment:
given this function doesn't really "send" but instead "shows" perhaps we name it `showEmailSuccessToast`?

Reply:
Gotcha, I meant to convey that it was the success(/error) toast for the send email button but that's clearer

Reply:
oooh. i see. in that case another option could have been `showSendEmailSuccessToast`. The first word always being the action/verb its triggering.

Reply:
its fine the way it is tho. no need to change anything.